snap/hooks/configure

```diff
@@ -371,4 +372,39 @@ then
     then
         chgrp microk8s -R "$SNAP_DATA/juju" "$SNAP_DATA/juju-home" || true
     fi
-fi
\ No newline at end of file
+fi
+
+# Set up a bridge with a known IP address that services can bind to instead of
+# whatever IP gets handed out by their router. Doesn't affect services that bind
+# to 127.0.0.1.
+API_IP="$(snapctl get api.ip)"
```

Comment:
Up until now we have not used the snapctl key-value store. The reason is that there are so many arguments that the user can change that we cannot have corresponding entries in the snapctl key value store. What we do instead is to have the directory $SNAP_DATA/args with files the user can change. Sure it is not a great experience but I am pretty sure the `snapctl get/set` experience will not be that great either.